userguide/content/en/docs/Adding content/diagrams-and-formulae/index.md

```diff
@@ -35,6 +35,13 @@ $$\tag*{(1)} P(E) = {n \choose k} p^k (1-p)^{n-k}$$
 The probability of getting \\(k\\) heads when flipping \\(n\\) coins is:
 $$\tag*{(1)}  P(E) = {n \choose k} p^k (1-p)^{n-k}$$
 
+You can also you [GLFM's math blocks](https://docs.gitlab.com/ee/user/markdown.html#math):
```

Comment:
This should read 'You can also **use**...', I guess!?

Reply:
Yes, fixed.